src/CommandPaletteCommand.js

```diff
@@ -4,10 +4,13 @@ import React from 'react';
 import CommandBlock from './CommandBlock';
 import AudioManager from './AudioManager';
 import { injectIntl } from 'react-intl';
+import type {IntlShape} from 'react-intl';
+
+export type CommandName = 'forward' | 'left' | 'right' | 'add' | 'deleteAll' | 'delete' | 'moveToPrevious' | 'moveToNext' | 'replace';
```

Comment:
Please see the comment on the usage of `CommandName` at `AudioManager.js`

Reply:
Done.